Cargo.toml

```
@@ -1,4 +1,47 @@
-[workspace]
-members = [
-  "fuel-bft"
+[package]
+name = "fuel-bft"
+version = "0.1.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+categories = ["cryptography::cryptocurrencies"]
+edition = "2021"
+homepage = "https://fuel.network/"
+keywords = ["blockchain", "cryptocurrencies", "fuel"]
+license = "Apache-2.0"
+repository = "https://github.com/FuelLabs/fuel-bft"
+description = "A Rust implementation of the Tendermint consensus protocol."
+
+[dependencies]
+async-trait = "0.1"
+fuel-crypto = { version = "0.4", default-features = false }
+fuel-types = { version = "0.3", default-features = false }
+rand = { version = "0.8", default-features = false, optional = true, features = [ "std_rng" ] }
+time = { version = "0.3", default-features = false }
+tokio = { version = "1.17", optional = true, features = [ "rt", "sync", "time" ] }
+tracing = { version = "0.1", default-features = false, optional = true }
+
+[dev-dependencies]
+rand = { version = "0.8", default-features = false, features = [ "std_rng" ] }
+tokio = { version = "1.17", features = [ "full" ] }
+
+[features]
+default = [
+    "fuel-crypto/default",
+    "fuel-types/default",
+    "memory",
+    "std",
+    "tokio-reactor"
 ]
+memory = [ "fuel-crypto/random", "rand", "std" ]
+std = [ "fuel-crypto/std", "fuel-types/std", "time/std" ]
```

`tracing` also has a std feature: https://github.com/tokio-rs/tracing/blob/master/tracing/Cargo.toml#L66

```suggestion
std = [ "fuel-crypto/std", "fuel-types/std", "time/std", "tracing/std" ]
```